api/opds.py

```diff
@@ -253,7 +253,15 @@ def visible_delivery_mechanisms(
         ).prioritize_for_pool(licensepool)
 
     def annotate_work_entry(
-        self, work, active_license_pool, edition, identifier, feed, entry, updated=None
+        self,
+        work,
+        active_license_pool,
+        edition,
+        identifier,
+        feed,
+        entry,
+        updated=None,
+        add_open_access_links=True,
```

Comment:
Is the new `add_open_access_links` parameter used anywhere in this function?

Reply:
Removed